drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/pom.xml

```diff
@@ -143,6 +143,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@psiroky Thanks Petr! To be honest I could really do with an explanation as to our logging configuration works - I still see some [log4j](https://github.com/droolsjbpm/drools-wb/blob/master/drools-wb-webapp/src/main/resources/log4j.xml) configuratrion files floating around...